package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Dec  7 14:26:51 UTC 2022 - David Diaz <dgonzalez@suse.com>
+
+- Allow reusing LVM volume groups
+- 4.5.15
```

Comment:
I wouldn't have added a new changelog entry but simply adding the version to the lastest one. In fact, now the last two entries are kind of contradictory :)

Reply:
You're right :( I didn't realize about it.